plugin.rb

```diff
@@ -36,6 +38,12 @@
   add_to_serializer(:user, :reminders_frequency) do
     RemindAssignsFrequencySiteSettings.values
   end
+
+  add_to_serializer(:group_show, :assignment_count) do
+    Topic.joins("JOIN topic_custom_fields tcf ON topics.id = tcf.topic_id AND tcf.name = 'assigned_to_id' AND tcf.value IS NOT NULL")
+      .where("tcf.value IN (SELECT group_users.user_id::varchar(255) FROM group_users WHERE (group_id IN (SELECT id FROM groups WHERE name = ?)))", object.name).count
+  end
+
```

Comment:
Looks good :+1:

We just need to make sure it is secure. This should only be included for users where `scope.can_assign?` is true. (see [here](https://github.com/discourse/discourse-assign/blob/master/plugin.rb#L237-L244) for a similar implementation)

Reply:
^^ I think this comment still needs to be addressed?